example/gardenadm-local/high-touch/namespace.yaml

```diff
@@ -2,3 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: gardenadm-high-touch
+  labels:
+    # Use shoot label so that machine pods are considered in export_artifacts() during test runs.
+    # This provides the usual systemd and pod logs for each machine.
```

Comment:
```suggestion
    # Use shoot label so that machine pods are considered in the export_artifacts() helper function during test runs in Prow.
    # This makes sure that the usual systemd and pod logs get exported for each machine pod in this namespace.
```